docs/pages/build-reference/caching.mdx

```diff
@@ -85,4 +85,24 @@ To using our CocoaPods cache server for your builds set the `EAS_BUILD_DISABLE_C
   }
   /* @hide ... */ /* @end */
 }
+```
+
+EAS Build allows you to cache the **Podfile.lock** file to provide consistent results across builds when using the managed workflow.
+To enable it, add `./ios/Podfile.lock` to the `"cache"."customPaths"` list in your build profile in **eas.json**.
```

Comment:
```suggestion
To enable it, add **./ios/Podfile.lock** to the `cache.customPaths` list in your build profile in **eas.json**.
```

I think we remove double quotes (it reads really weird and not consistent with rest of the docs when we specify properties defined in JSON files) and also make the [filename bold as per our writing styleguide](https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md#filenames-as-bold-text).